src/dotnet/commands/dotnet-new/LocalizableStrings.cs

```diff
@@ -0,0 +1,35 @@
+namespace Microsoft.DotNet.Tools.New
+{
+    internal static class LocalizableStrings
+    {
+        public static const string AppName = "dotnet new";
+
+        public static const string AppFullName = ".NET Initializer";
+
+        public static const string AppDescription = "Initializes empty project for .NET Platform"; 
+
+        public static const string CreateFailedBecauseProjectExists = "Creating new {0} project failed, project already exists.";
+
+        public static const string CreateFailedBecauseContentExists = "Creating new {0} project failed, directory already contains {1}";
+
+        public static const string CreateFailed = "Creating new {0} project failed.";
```

Comment:
When I look only on this string I believe it is hard to read the context how to translate this sentence (I wouldn't know how to translate this to Polish without more context or some example word I can put there) - I think it should be something like {ProgrammingLanguageName} or at least a comment explaining: {0} -> i.e. "C#"